src/Themes.ts

```diff
@@ -1,12 +1,15 @@
-import {redrawProof, redrawTree} from "./SharedToolUtils/DrawUtils";
-import {TreeContext} from "./TreeContext";
-
 /**
  * Provides utilities for extracting important colors for the application from the CSS themes.
- *
  * @author James Oswald
  */
 
+// Import and initialize the theme-change library.
+import {themeChange} from "theme-change";
```

Comment:
Would prefer the import statements be alphabetized according to the first letter of the imported object or method